controllers/articles.controllers.js

```diff
@@ -38,13 +43,19 @@ exports.getCommentsByArticleId = (req, res, next) => {
 };
 
 exports.postCommentToArticle = (req, res, next) => {
-    const {article_id} = req.params;
-    const {body} = req
-    addCommentToArticle(article_id, body)
-    .then((comment) => {
-        res.status(201).send({comment})
+  const { article_id } = req.params;
+  const { body } = req;
+  const { username } = body
+  const checkArticle = checkArticleExists(article_id)
+  const checkUser = checkUserExists(username)
```

Comment:
You don't need an extra check on the user - author in the comments table references username in the users table - it's a foreign key. 

If it doesn't exist in the users table SQL will error